lib/scala/text-buffer/src/main/java/org/enso/text/Hex.java

```diff
@@ -0,0 +1,18 @@
+package org.enso.text;
+
+/**
+ * A utility class that serves as a replacement of some methods from {@code org.bouncycastle}, so
+ * that we don't have to depend on it.
+ */
+public final class Hex {
```

Comment:
Maybe you don't want to make the class `public`.

Reply:
It is used both from within `text-buffer` and also from `runtime`. It is sort of a minified replacement of `org.bouncycastle.Hex`.